### PR TITLE
Reduce the image size by using debian:jessie-slim

### PR DESCRIPTION
**Pull request by robbyoconnor**
Your image size is absurdly large. I rather than doing an immediate PR -- based work on it (attributing back of course). 
This also  contains some micro-optimizations to take advantage of layer caching for things that don't change. Lines 17 onward are the things that change. Everything else is the same.

### CODE REVIEW

**Comment by robbyoconnor** [comment]
The overall image size on mine is [51 MB compressed; 157 MB uncompressed](https://hub.docker.com/r/robbyoconnor/dockup/tags/) vs. [157 MB compressed; 388 MB uncompressed](https://hub.docker.com/r/wetransform/dockup/tags/)

File: Dockerfile [hunk 1]
```diff
@@ -18,6 +14,14 @@ ENV NOTIFY_BACKUP_SUCCESS false
 ENV NOTIFY_BACKUP_FAILURE false
 ENV BACKUP_TAR_TRIES 5
 ENV BACKUP_TAR_RETRY_SLEEP 30
-
-WORKDIR /dockup
```
Comment (stempler) [inline Dockerfile:22]: Any specific reason not to add the workdir again? I would like to keep it compatible to the behavior before.

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,7 @@
...
-MAINTAINER Simon Templer <simon@wetransform.to>
-
-RUN apt-get update && apt-get install -y python-pip curl && pip install awscli
-
-ADD /scripts /dockup/
-RUN chmod 755 /dockup/*.sh
-
+FROM debian:jessie-slim
+LABEL maintainer="Simon Templer <simon@wetransform.to>"
+ENV CRON_TIME="0 0 * * *"
```
Comment (stempler) [inline Dockerfile:3]: I would prefer to leave that out to keep the default behavior as it was before.

**Review by stempler — COMMENTED** [review]
Regarding the image size I must admit I did not really care to change anything from the original image, thanks for the optimizations. I only have some minor comments in the code.

File: Dockerfile (hunk 2 above)
Comment (robbyoconnor) [reply Dockerfile:3]: eh, that's doable. This is still overriable.

File: Dockerfile (hunk 1 above)
Comment (robbyoconnor) [reply Dockerfile:22]: That might not have been intentional..

**Comment by robbyoconnor** [comment]
Re: the size -- I care a lot. It was unnecessarily bloated.

**Pull request merged**